graphium/cli/train_finetune_test.py

```diff
@@ -70,15 +70,12 @@ def get_replication_factor(cfg):
 
 
 def get_gradient_accumulation_factor(cfg):
+    """
+    WARNING: This MUST be called after accelerator overrides have been applied
```

Comment:
Can you specify what is the `accelerator overrides`? Is this a function? Where is it?

Reply:
These are applied in `load_accelerator`, I'll clarify the comment